benchmarks/src/main/scala/io/finch/benchmarks.scala

```diff
@@ -13,13 +15,18 @@ import shapeless._
 
 import java.nio.charset.{Charset, StandardCharsets}
 import java.util.concurrent.{ThreadLocalRandom, TimeUnit}
+import scala.concurrent.Future
 
 @BenchmarkMode(Array(Mode.AverageTime))
 @OutputTimeUnit(TimeUnit.NANOSECONDS)
 @Warmup(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
 @Measurement(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
 @Fork(2)
 abstract class FinchBenchmark extends Endpoint.Module[IO] {
+  implicit val dispatcher: Dispatcher[IO] = new Dispatcher[IO] {
```

Comment:
Isn't there an instance of a Dispatcher for IO already?

Reply:
Yes there is but it returns a `Resource`